AmahiAnywhere/AmahiAnywhere/MenuViewController.swift

```diff
@@ -0,0 +1,178 @@
+                self.navigationController?.popToRootViewController(animated: true)
+            })
+            self.present(myalert, animated: true)
+        case .sent:
+            let myalert = UIAlertController(title: StringLiterals.SENT_TITLE, message: StringLiterals.SENT_MESSAGE, preferredStyle: UIAlertControllerStyle.alert)
+            
+            myalert.addAction(UIAlertAction(title: StringLiterals.ALERT_ACTION, style: .default) { (action:UIAlertAction!) in
+                self.navigationController?.popToRootViewController(animated: true)
+            })
+            self.present(myalert, animated: true)
+            
+        case .failed:
+            print("Mail sent failure: \(String(describing: error?.localizedDescription))")
+            // default:
+            break
+        }
+        self.dismiss(animated: true, completion: nil)
+    }
+    // MARK:- Signout function Added
+    
+    func signOut()
+    {
+        LocalStorage.shared.logout {}
+        let loginVc = self.instantiateViewController(withIdentifier: "LoginViewController" ,from: StoryBoardIdentifiers.MAIN)
+        self.present(loginVc, animated: true, completion: nil)
+    }
+    // MARK:- Delegates Method
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return ManuNameArray.count
+        
+    }
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "MenuTableViewCell", for: indexPath) as! MenuTableViewCell
+        
+        
+        
+        cell.lblMenuName.text! = ManuNameArray[indexPath.row]
+        cell.lblMenuName.textColor = UIColor.white
+        
+        return cell
+    }
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        
+        tableView.deselectRow(at: indexPath, animated: true)
+        let revealviewcontroller:SWRevealViewController = self.revealViewController()
+        
+        let cell:MenuTableViewCell = tableView.cellForRow(at: indexPath) as! MenuTableViewCell
+        
+        if cell.lblMenuName.text! == "About"
+        {
+            let myalert = UIAlertController(title: StringLiterals.ABOUT_TITLE, message: StringLiterals.ABOUT_MESSAGE, preferredStyle: UIAlertControllerStyle.alert)
+            
+            myalert.addAction(UIAlertAction(title: StringLiterals.ALERT_ACTION, style: .default) { (action:UIAlertAction!) in
+                self.navigationController?.popToRootViewController(animated: true)
+            })
+            self.present(myalert, animated: true)
+            
+            
+        }
+        if cell.lblMenuName.text! == "Rate"
+        {
+            
+            
+            let appID = StringLiterals.APP_ID
+            let urlStr = "itms-apps://itunes.apple.com/app/viewContentsUserReviews?id=\(appID)"
```

Comment:
You can move this URL to StringLiterals and mark it with something like `User Review URL`